examples/src/main/java/io/dapr/examples/OpenTelemetryConfig.java

```diff
@@ -57,13 +62,45 @@ public static Tracer createTracer(String instrumentationName) {
               .setServiceName(InvokeClient.class.getName())
               .build();
 
-      OpenTelemetrySdk.getGlobalTracerManagement()
-          .addSpanProcessor(SimpleSpanProcessor.builder(zipkinExporter).build());
+      SdkTracerProvider sdkTracerProvider = SdkTracerProvider.builder()
+          .addSpanProcessor(SimpleSpanProcessor.create(zipkinExporter))
+          .build();
+
+      return OpenTelemetrySdk.builder()
+          .setTracerProvider(sdkTracerProvider)
+          .setPropagators(ContextPropagators.create(W3CTraceContextPropagator.getInstance()))
+          .buildAndRegisterGlobal();
     } else {
       System.out.println("WARNING: Zipkin is not available.");
     }
 
-    return tracer;
+    return null;
+  }
+
+  /**
+   * Converts current OpenTelemetry's context into Reactor's context.
+   * @return Reactor's context.
+   */
+  public static reactor.util.context.Context getReactorContext() {
+    return getReactorContext(Context.current());
+  }
+
+  /**
+   * Converts given OpenTelemetry's context into Reactor's context.
+   * @param context OpenTelemetry's context.
+   * @return Reactor's context.
+   */
+  public static reactor.util.context.Context getReactorContext(Context context) {

```

Comment:
This logic can potentially be exported into a separate package to be published: sdk-opentelemetry. It would contain utility classes to use opentelemetry with dapr. We can do that once opentelemetry is stable.